web/src/data/authProviders.js

```diff
@@ -89,15 +89,33 @@ const authProviders = [
     platform: true,
     selfHosted: true,
   },
-  // {
-  //   name: 'MessageBird',
-  //   // logo: '/img/libraries/dart-icon.svg',
-  //   href: '/docs/guides/auth/auth-twilio',
-  //   official: false,
-  //   supporter: 'Messagebird',
-  //   platform: false,
-  //   selfHosted: true,
-  // },
+  {
+    name: 'MessageBird',
+    // logo: '/img/libraries/dart-icon.svg',
+    href: 'https://github.com/supabase/gotrue/pull/210',
```

Comment:
Help wanted: should we create a guide for this?

Looking at the [twilio one](https://supabase.io/docs/guides/auth/auth-twilio), it might be a lot of work for this month's release

Reply:
i can work on this, messagebird's set up is slightly less complicated and i had the chance to try it out when testing the PR